corn/simpleBlogHomePage/styles.css

```diff
@@ -0,0 +1,193 @@
+:root {
+  --h2-font-family: "Kanit", serif;
+  --html-background-color: #fff;
+  --rocket-size: 48px;
+  --rocket-width: var(--rocket-size);
+  --rocket-height: var(--rocket-size);
+  --front-end-optimizing-css-font-family: "Playfair Display", serif;
+  --optimizing-css-color: #ffffff;
+  --border: 1px solid #f5f5f5;
+  --read-full-text-font-family: "Roboto", serif;
+  --text-ago-text-color: #6e6e6e;
+  --front-end-color: #4b4b4b;
+}
+
+.container,
+.container--css-variables {
+  position: relative;
+}
+
+.container {
+  height: 296px;
+  margin-top: 48px;
+  padding: 8px;
+}
+
+.container,
+.header-padding,
+.position {
+  box-sizing: border-box;
+}
+
+.container,
+html,
+.position-thee-card {
+  width: 1144px;
+}
+
+.header-padding,
+.position {
+  padding: 10px;
+}
+
+a {
+  text-decoration: none;
+}
+
+h2 {
+  font-family: var(--h2-font-family);
+  margin: 8px 0px;
+}
+
+html {
+  height: 100px;
+  margin: 0 auto;
+  background-color: var(--html-background-color);
+}
+
+.rocket,
+.optimizing-css {
+  position: absolute;
+}
+
+.rocket,
+.text-big,
+.front-end-small {
+  display: flex;
+}
+
+.rocket {
+  width: var(--rocket-width);
+  height: var(--rocket-height);
+  left: 24px;
+  top: 64px;
+  line-height: 57px;
+  align-items: center;
+  font-size: 200%;
+}
+
+.front-end,
+.optimizing-css {
+  font-family: var(--front-end-optimizing-css-font-family);
+}
+
+.optimizing-css {
+  width: 434px;
```

Comment:
It is better to use other approaches and don't use exact width for just a header